libs/screen/targetoverrides.ts

```diff
@@ -10,8 +10,8 @@ const screen = image.create(DAL.LCD_WIDTH, DAL.LCD_HEIGHT)
 namespace _screen_internal {
     //% shim=pxt::updateScreen
     function updateScreen(img: Image): void { }
-    //% shim=pxt::updateStats
     function updateStats(msg: string): void { }
+    //% shim=pxt::updateStats
```

Comment:
```suggestion
    //% shim=pxt::updateStats
    function updateStats(msg: string): void { }
```

not sure why the merge conflict resolution ui moved this but just putting it back so I can merge